auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -470,17 +345,17 @@ export default defineComponent({
       RESUBMITTED: 'Resubmitted',
       REJECTED: 'Rejected',
       APPROVED: 'Approved',
-      ABANDONED: 'Abandoned'
+      ABANDONED: 'Abandoned',
+      CANCELLED: 'Cancelled'
     }
 
     const state = reactive({
       reviews: [] as Array<ContinuationReviewIF>,
       headers: [
         { text: 'Date Submitted', value: 'submissionDate' },
-        { text: 'Future Effective Date', value: 'futureEffectiveDate' },
         { text: 'NR Number', value: 'nrNumber' },
         { text: 'Identifying Number', value: 'identifier' },
-        { text: 'Completing Party', value: 'completingParty' },
+        { text: 'Registered Office', value: 'officeEmail' },
```

Comment:
This is another pending design change, that I think it worth implementing as-is until we hear more about this.

Reply:
You could still use `completingParty` value field from backend by only changing text.

Reply:
I mean for now (since design change is pending), i dont want to changes until its confirmed.

Reply:
OK, then I will set the value to be completingParty for now

Reply:
Wait. If we change this then there's a chance we won't need to come back and change it. Whereas if we leave this then it's guaranteed we'll have to come back here and update it.

Reply:
OK, fine, just leave what you have for now.

Reply:
Resolving.